python/ray/util/rpdb.py

```diff
@@ -190,26 +191,35 @@ def connect_ray_pdb(host=None,
                     port=None,
                     patch_stdstreams=False,
                     quiet=None,
-                    breakpoint_uuid=None):
+                    breakpoint_uuid=None,
+                    debugger_external=False):
     """
     Opens a remote PDB on first available port.
     """
-    if host is None:
+    if debugger_external:
+        assert not host, "Cannot specify both host and debugger_external"
+        host = "0.0.0.0"
+    elif host is None:
         host = os.environ.get("REMOTE_PDB_HOST", "127.0.0.1")
```

Comment:
should we print a warning or something if REMOTE_PDB_HOST is set and the debugger_external flag is passed?